pkg/cmd/prebuild/update.go

```diff
@@ -19,10 +18,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	branchFlag         string
+	retentionFlag      int
+	commitIntervalFlag int
+	triggerFilesFlag   []string
+	projectConfigFlag  string
+	runOnUpdateFlag    bool
+)
+
 var prebuildUpdateCmd = &cobra.Command{
-	Use:   "update",
+	Use:   "update [project-config] [prebuild-id]",
```

Comment:
Let's use `[PROJECT_CONFIG]` and `[PREBUILD_ID]` to be consistent with other commands

Reply:
Updated